src/nrc/templates/maykin_2fa/base.html

```diff
@@ -0,0 +1,9 @@
+{% extends "maykin_2fa/base.html" %}
+
+{# Django 3.2 #}
+{% block breadcrumbs %}{% endblock %}
+
+{# Do not show any version information #}
+{% block footer %}
+    <div id="footer"></div>
+{% endblock %}
```

Comment:
I think this file can be removed, we're already on Django 4.2 and I don't think it's picked up by `maykin_2fa/login.html` anyway